subprojects/core/src/integTest/groovy/org/gradle/initialization/CalculateTaskGraphBuildOperationIntegrationTest.groovy

```diff
@@ -81,6 +81,28 @@ class CalculateTaskGraphBuildOperationIntegrationTest extends AbstractIntegratio
         operation().result.excludedTaskPaths == []
     }
 
+    @ToBeFixedForInstantExecution
```

Comment:
Please check with the instant execution team what is necessary to make this work.

Reply:
double checked with paul about those tests